divi/src/main.h

```diff
@@ -240,6 +240,7 @@ void Misbehaving(NodeId nodeid, int howmuch);
 /** Flush all state, indexes and buffers to disk. */
 void FlushStateToDisk();
 
+const std::string strMessageMagic = "DarkNet Signed Message:\n";
 
```

Comment:
Given the importance of this variable to signature verification, it should be a static member of CObfuscationSigner

Reply:
i would like to address this in another pr